.github/workflows/deploy_app_function.yaml

```diff
@@ -24,7 +25,7 @@ on:
 jobs:
   deploy:
     name: Deploy
-    uses: pagopa/dx/.github/workflows/function_app_deploy.yaml@f2c7b99b6340760a3fe6727f0f3fd0fe96d65618
+    uses: pagopa/dx/.github/workflows/function_app_deploy.yaml@cf980ca76172b71629f6c76ba34f74d07c991d05
```

Comment:
**note**: this sha is not yet merged in main.

This is related to the PR https://github.com/pagopa/dx/pull/17